tests/conftest.py

```diff
@@ -0,0 +1,174 @@
+import uuid
+from datetime import datetime, UTC
+
+import pytest
+
+from dor.domain.models import Bin
+from dor.providers.models import (
+    Agent, AlternateIdentifier, FileMetadata, FileReference, PackageResource,
+    PreservationEvent, StructMap, StructMapItem, StructMapType
+)
+
+
+@pytest.fixture
+def sample_bin() -> Bin:
+    return Bin(
+        identifier=uuid.UUID("00000000-0000-0000-0000-000000000001"), 
+        alternate_identifiers=["xyzzy:00000001"], 
+        common_metadata={
+            "@schema": "urn:umich.edu:dor:schema:common",
```

Comment:
With `pytest`, you can define shared fixtures in `conftest.py` and then use them in test files by using the `pytest.mark.usefixtures` decorator (see https://docs.pytest.org/en/stable/how-to/fixtures.html#use-fixtures-in-classes-and-modules-with-usefixtures). I elected to move `sample_bin` here since were were using it in three places. I can see there being downsides to this approach, so let me know what you think.